servicetests/tests/ShadowUpdate/main.cpp

```diff
@@ -0,0 +1,331 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+#include <aws/crt/Api.h>
+#include <aws/crt/JsonObject.h>
+#include <aws/crt/UUID.h>
+#include <aws/crt/io/HostResolver.h>
+
+#include <aws/crt/mqtt/Mqtt5Packets.h>
+#include <aws/iot/Mqtt5Client.h>
+#include <aws/iot/MqttClient.h>
+
+#include <aws/iotshadow/ErrorResponse.h>
+#include <aws/iotshadow/IotShadowClient.h>
+#include <aws/iotshadow/ShadowDeltaUpdatedEvent.h>
+#include <aws/iotshadow/ShadowDeltaUpdatedSubscriptionRequest.h>
+#include <aws/iotshadow/UpdateNamedShadowRequest.h>
+#include <aws/iotshadow/UpdateShadowRequest.h>
+#include <aws/iotshadow/UpdateShadowResponse.h>
+#include <aws/iotshadow/UpdateShadowSubscriptionRequest.h>
+
+#include <aws/iotshadow/GetShadowRequest.h>
+#include <aws/iotshadow/GetShadowResponse.h>
+#include <aws/iotshadow/GetShadowSubscriptionRequest.h>
+#include <aws/iotshadow/UpdateShadowSubscriptionRequest.h>
+
+#include <algorithm>
+#include <condition_variable>
+#include <iostream>
+#include <mutex>
+#include <thread>
+
+#include "../../../samples/utils/CommandLineUtils.h"
+
+using namespace Aws::Crt;
+using namespace Aws::Iotshadow;
+
+void changeShadowValue(Aws::Crt::String thingName, String property, String value);
+void changeNamedShadowValue(String thingName, String property, String value, String shadowName);
+
+IotShadowClient *shadowClient = nullptr;
+int main(int argc, char *argv[])
+{
+    printf("Starting shadow update test\n");
```

Comment:
There's a lot of debug-esque output throughout here.  It should either be removed (if it's not necessary) or made into logging statements so that CRT logs from runs give a full picture of what's going on.